test/test-app/config/connections.js

```diff
@@ -18,11 +18,13 @@ module.exports = {
 
     two: {
         adapter: adapterTwo,
-        config: {
-            url: 'https://localhost:2;localhost:3;localhost:4',
-            user: 'user',
-            password: 'password',
-            database: 'test'
+        config: function(mycro) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["mycro" is defined but never used](https://www.codacy.com/app/chris-ludden/mycro/file/1988801754/issues/source?bid=3045492&fileBranchId=3118985#l21)